config/replace.config.js

```diff
@@ -1,147 +1,31 @@
-const modifiedFile = '// This file has been added and patched after installing the NPM modules (via NPM script \'prepare\')\n';
-const importTHREE = 'import * as THREE from \'three\';\n';
-const declareNewSpaceName = `const ${spaceName} = {};\n`;
-patchs.push(() => replace({
-    files,
-    from: /^(.*)$/m,
-    to: match => `${modifiedFile}${importTHREE}${declareNewSpaceName}${match}`,
-}));
-
-// Export default function in ending file
-patchs.push(() => replace({
-    files,
-    from: /.*$/i,
-    to: (match, n, pat, file) => {
-        const basename = path.basename(file, '.js');
-        return `\nexport default ${spaceName}.${basename};\n`;
-    },
-}));
-
-// Export alls other functions in ending file
-patchs.push(() => replace({
-    files,
-    from: /.*$/i,
-    to: (match, n, pat, file) => {
-        const basename = path.basename(file, '.js');
-        if (exportTables[basename] && exportTables[basename].size) {
-            exportTables[basename].toString();
-            return `export {${[...exportTables[basename]].join()}};\n`;
-        }
-        return match;
-    },
-}));
-
-// Add import from threeExtented
-patchs.push(() => replace({
-    files,
-    from: declareNewSpaceName,
-    to: (match, n, pat, file) => {
-        const basename = path.basename(file, '.js');
-        if (match == declareNewSpaceName && importTables[basename].size) {
-            return match + [...importTables[basename]].map((n) => {
-                const from = path.dirname(filesExamples.find(f => path.basename(f, '.js') === basename)).replace('.', '');
-                const to = path.dirname(filesExamples.find(f => path.basename(f, '.js') === n)).replace('.', '');
-                const relative = path.relative(`_/${from}`, `_/${to}`).replace('\\', '\/');
-                if (relative == '') {
-                    return `import ${n} from './${n}';`;
-                } else {
-                    return `import ${n} from '${relative}/${n}';`;
-                }
-            }).join('\n');
-        } else {
-            return match;
-        }
-    },
-}));
-
 // Patch _BATCHID in GLTFLoader
-const fileGLTFLoader = files.filter(f => path.basename(f, '.js') === 'GLTFLoader');
 const patchBatchID = /JOINTS_0: +'skinIndex',/g;
 const patchGLTFLoader = 'JOINTS_0: \'skinIndex\',\n\t\t_BATCHID: \'_BATCHID\',';
 
-patchs.push(() => replace({
+const starting_promise = replace({
     files: fileGLTFLoader,
     from: patchBatchID,
     to: patchGLTFLoader,
-}));
-
-// Patch _BATCHID in LegacyGLTFLoader
-const fileLegacyGLTFLoader = files.filter(f => path.basename(f, '.js') === 'LegacyGLTFLoader');
-const fromBatchIDLegacy = /case +'JOINT':\s+geometry\.addAttribute\( 'skinIndex', bufferAttribute \);\s+break;/g;
-const toLegacyGLTFLoader = 'case \'JOINT\':\ngeometry.addAttribute( \'skinIndex\', bufferAttribute );\nbreak;\ncase \'_BATCHID\':\ngeometry.addAttribute( \'_BATCHID\', bufferAttribute );\nbreak;';
+});
 
 patchs.push(() => replace({
```

Comment:
No need to use `patchs` anymore if it is the only one in it.

Reply:
done